app/services/notification_service.rb

```diff
@@ -0,0 +1,53 @@
+class NotificationService
+  DEFAULT_LOCALE = :en
+
+  def client
+    @client ||= Twilio::REST::Client.new(twilio_account_sid, twilio_auth_token)
+  end
+
+  def send_sms(recipient_number, message, callback_url=nil)
```

Comment:
This seems incorrect. The OTP SMSes (from `SendPatientOtpSmsJob#send_sms`) use a different twilio account than the reminder messages since that's the only way to segregate the short-codes (`BPCALL` etc.). The original `SmsNotification` class would optionally take in the client itself so you could segregate accounts cleanly (and/or pass in a different client which is not Twilio-related)

We should also do that here in some fashion, a dumb way to do it would be to create two clients. But I prefer passing in the Client because it makes the `NotificationService` mechanism-agnostic (a bit).

Reply:
Another way to do this could be to create a `MessagingClient` abstraction that returns the client that you need without having to worry about `sid`, `auth_token` details.

Reply:
Ah interesting. So, the notification service can't limit its awareness to just recipient number and message content, because different messages need to come from different senders? Is the sender number sufficient info, or is there a completely different set of credentials for each sender?

Reply:
Yeah, as far as I know. A single `TwilioClient` holds 1 account at a time which involves two things: the auth token and the account id, we have 2 different accounts (currently).

Reply:
We could just optionally pass these credentials into the `NotificationService` initializer. I really, really don't want to put more refactoring into this PR if we can avoid it.